src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,137 @@
 package com.theironyard.novauc;
 
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Main {
 
-    public static void main(String[] args) {
-	// write your code here
+    public static Scanner fetchItem = new Scanner(System.in);
+    public static Scanner scanner = new Scanner(System.in);
+    public static ArrayList<Item> inventory = new ArrayList<>();
+
+
+    public static void main(String[] args) throws Exception {
+
+        //System.out.println("Choose what equipment you would like to spawn: \n");
+
+        //System.out.println("what is the name of your item?");
+        inventory.add(createItem("Sword of Fire", 1, "weapon"));
+        inventory.add(createItem("Bow of faulty aim", 1, "weapon"));
+        inventory.add(createItem("The Breast Plate of Milk", 1, "armor"));
+        inventory.add(createItem("The Frivilous Cardboard Shield of Death", 1, "armor"));
+        inventory.add(createItem("arrow", 1, "MiscItem"));
+        inventory.add(createItem("money", 1, "MiscItem"));
+        inventory.add(createItem("Shirt", 1, "Clothing"));
+        inventory.add(createItem("Medallion", 1, "Jewelery"));
+//        int j = 1;
+//        for(Item item: inventory) { //this for loop goes through every item in inventory so that we can print them out later
+//            System.out.println(j + ". " + item.toString());
+//            j = j + 1;
+//        }
+        for(int i=0; i < inventory.size(); i++) {  //this for loop does the same thing as the one above but we an see it counting/depicting the inventory size and adding to it as it loops
+            System.out.println(i+1 + ". " + inventory.get(i).toString());
+        }
+
+        //ask the user for a variable input that is equal to the category
+        //have that return the string name
+        //Create a static `createItem` method in your main class which returns an object using the correct category class (or throws and error for an invalid category name).
     }
+
+
+    public static Item createItem(String name, int quantity, String category) throws Exception {
+        Item x = null;
+
+        switch (category.toLowerCase()) {
+            case "weapon":
+                x = new Weapon(name, quantity);
+                //inventory.add((Item) sword);//this is a cast which liek casts a spell to
+                break;
+            case "clothing":
```

Comment:
Could have had a return statement in each case instead of setting the new object to x.  Example line 46 would read return new Weapon(name, quantity); The way you did it accomplishes the same task.  So no big deal.  Just a bit cleaner with the return statement in each case.  Good job.